app/src/main/res/values/integers.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <integer name="reservation_date_picker_start_year">2021</integer>
```

Comment:
what happens if we pass the year and we are in 2022?

Reply:
was not necesary, deleted